rps.c

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+/*rock paper scissors with drop-down menu
+  @author Aly Milich
+  */
+
+int main(){
+
+	int choice, computer, tries = 0, rock =1, paper = 2, scissors = 3, yourscore = 0, compscore =0;
+
+	do{
+
+		printf("Press 1 for rock, 2 for paper, and 3 for scissors.\n");
+		scanf("%d", &choice);
+		srand(time(NULL));
+		int computer = rand()%4;
```

Comment:
This is your one harmful mistake! Remember from last assignment that this would return a range of 0-3, rather than 1-3. To fix this, you should have used rand()%3 + 1. Besides that, your continuous if statements could have been else ifs instead.